Description: Avoid to use cmd for calculation of paths
  We are simulating a mingw environment, but in some cases it is better to
  remain in unix e.g. when it comes to path names.

Author: Laszlo Boszormenyi (GCS) <gcs@debian.org>
Bug-Debian: https://bugs.debian.org/776821
Bug-Debian: https://bugs.debian.org/804026
Bug-Debian: https://bugs.debian.org/813338

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Last-Update: 2016-07-30

--- icu-57.1.orig/source/config/mh-mingw
+++ icu-57.1/source/config/mh-mingw
@@ -95,10 +95,13 @@ LIBICU = $(STATIC_PREFIX_WHEN_USED)$(ICU
 # The #M# is used to delete lines for icu-config
 # Current full path directory.
 #CURR_FULL_DIR=$(shell pwd -W)#M# for MSYS
-CURR_FULL_DIR?=$(subst \,/,$(shell cmd /c cd))#M# for Cygwin shell
+#CURR_FULL_DIR?=$(subst \,/,$(shell wine cmd /c cd))#M# for Cygwin shell
 # Current full path directory for use in source code in a -D compiler option.
 #CURR_SRCCODE_FULL_DIR=$(subst /,\\\\,$(shell pwd -W))#M# for MSYS
-CURR_SRCCODE_FULL_DIR=$(subst \,/,$(shell cmd /c cd))#M# for Cygwin shell
+#CURR_SRCCODE_FULL_DIR=$(subst \,/,$(shell wine cmd /c cd))#M# for Cygwin shell
+CURR_FULL_DIR?=$(shell pwd | sed 's/ /\\ /g')
+# Current full path directory for use in source code in a -D compiler option.
+CURR_SRCCODE_FULL_DIR?=$(shell pwd | sed 's/ /\\ /')
 
 ## Compilation rules
 %.$(STATIC_O): $(srcdir)/%.c
